qrcode/qrcode.go

```diff
@@ -1,182 +0,0 @@
-package qrcode
```

Comment:
Ah this is nice, moving code directly related to its respective domain, into that domain / package. It is also a good way of preventing circle dependencies.